docs/operators/transform.md

```diff
@@ -0,0 +1,167 @@
+Transform - Operator Documentation
+==================================
+
+About Transform operator
```

Comment:
Can you make this title type similar to "Configuration Parameter" i.e. with ####